addons/medical_gui/initSettings.sqf

```diff
@@ -108,3 +108,11 @@ private _categoryColors = [ELSTRING(medical,Category), format ["| %1 |", LELSTRI
         false // isGlobal
     ] call CBA_fnc_addSetting;
 } forEach _damageColors;
+
+[
+    QGVAR(showBloodlossEntry),
+    "CHECKBOX",
+    [LSTRING(ShowBloodlossEntry_DisplayName), LSTRING(ShowBloodlossEntry_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    true
+] call CBA_fnc_addSetting;
```

Comment:
Unsure on this but maybe we want to force this to be a global setting.

Reply:
Personally I think settings should only be global where absolutely necessary. There seems no reason not to just leave this up to server admins.